macvtap_test.go

```diff
@@ -121,6 +119,7 @@ var _ = Describe("macvtap Operations", func() {
 			Netns:       targetNs.Path(),
 			IfName:      IFNAME,
 			StdinData:   []byte(conf),
+			Args:        fmt.Sprintf("MAC=%s", macAddress),
```

Comment:
Eeeeh, still. What if this containers PETR=FOO and no MAC?

Reply:
Better now ?

Reply:
Whatchu mean? I don't see such a test.

Reply:
What I miss is a test that provides Args, but none of the args is MAC. So it would require another test case with `Args:        fmt.Sprintf("PETR=FOO"),`

Reply:
I'll add such a test; never the less, this makes me wonder: should the CNI plugin disregard that unknown argument, or fail ? 

Pretending stuff didn't happen might count as a coping mechanism, but here ...